EventsExpress/ClientApp/src/components/helpers/form-helpers/location-map-with-marker.jsx

```diff
@@ -5,18 +5,17 @@ import { LocationMap } from '.';
 
 export default function LocationMapWithMarker(props) {
     let initialPos = { lat: 50.4547, lng: 30.5238 };
-    if (props.input.value.latitude != undefined) {
-        initialPos = { lat: props.input.value.latitude, lng: props.input.value.longitude };
+    if (props.latitude != null) {
+        initialPos = { lat: props.latitude, lng: props.longitude };
```

Comment:
It would be better to use `Object.assign()` here to merge with defaults. 

see detailed explanations [here|https://github.com/ryanmcdermott/clean-code-javascript#set-default-objects-with-objectassign]